docs/modules/ROOT/pages/explanations/decisions/subscription-tracking.adoc

```diff
@@ -0,0 +1,71 @@
+= Subscription tracking
+
+== Problem
+
+We need to track the yearly subscriptions bought for OpenShift 4 clusters.
+Subscription count is required for a monthly report to RedHat.
+RedHat subscriptions are based on (v)CPU count.
+We need to cover the difference of the yearly subscription count and the actual CPU count with monthly subscriptions.
+
+Buying yearly subscriptions saves us money compared to buying monthly subscriptions.
+
+It also adds a small amount of risk if a customer decides to cancel their clusters before the year is over.
+
+=== Goals
+
+* Yearly subscriptions are tracked in a central place
+* Subscription count is available for an automated monthly report to RedHat
+* A suggestion for how many yearly subscriptions to buy is available to the account manager
+
+=== Non-goals
+
+* Full automation of subscription buying
+
+== Proposals
+
+=== Option 1: Database and Web UI (custom/ Grafana)
+
+We store subscriptions in a database and build a custom UI or Grafana dashboard to visualize the data.
+
+This allows instant feedback and would allow account managers to track subscriptions themselves without having to ask our team.
+
+Building the UI would take time and we would need to maintain it.
+
+Grafana most likely doesn't support editing data.
+
+=== Option 2: Store in ConfigMap managed by Commodore
```

Comment:
I don't understand this option. It only talks about where to store data, but how can this data be accessed? Option 1 clearly states where to store and how to work with the data, this option here seems to lack more profound information about that.

Reply:
I added a process proposal.